packages/fxa-content-server/app/scripts/lib/experiments/grouping-rules/key-stretch.js

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const BaseGroupingRule = require('./base');
+
+const GROUPS = [
+  'control',
+  'v2',
+];
+
+const ROLLOUT_RATE = 0.0;
```

Comment:
Would be nice if we can control this with an env var.

Reply:
Ah good call. Meant to add that in.

Reply:
Seems like none of the other rules in content server use config... I've filed a follow up for this, FXA-9080. When we turn this on, it'd definitely be nice to be able to adjust without a code change.